src/System.IdentityModel.Tokens.Jwt/JwtPayload.cs

```diff
@@ -72,16 +72,17 @@ internal static JwtPayload CreatePayload(byte[] bytes, int length)
                     if (reader.ValueTextEquals(JwtPayloadUtf8Bytes.Aud))
                     {
                         reader.Read();
-                        payload._audiences = new List<string>();
+                        payload._audiences = new();
                         if (reader.TokenType == JsonTokenType.StartArray)
                         {
                             JsonSerializerPrimitives.ReadStrings(ref reader, payload._audiences, JwtRegisteredClaimNames.Aud, ClassName, false);
                             payload[JwtRegisteredClaimNames.Aud] = payload._audiences;
                         }
                         else
                         {
-                            payload._audiences.Add(JsonSerializerPrimitives.ReadString(ref reader, JwtRegisteredClaimNames.Aud, ClassName, false));
-                            payload[JwtRegisteredClaimNames.Aud] = payload._audiences[0];
+                            string audStr = JsonSerializerPrimitives.ReadString(ref reader, JwtRegisteredClaimNames.Aud, ClassName, false);
```

Comment:
Why it is necessary to create the local string: 'audStr'?

Reply:
It is used twice below

Reply:
It isn't strictly necessary, removed